.npm/package.json

```diff
@@ -4,7 +4,7 @@
   "description": " Set up a local development environment for COMIT apps with one command.",
   "main": "cca.js",
   "scripts": {
-    "start": "node cca.js",
+    "start": "node cca.js new",
```

Comment:
📣 this does not work. As per [`yarn create` documentation](https://yarnpkg.com/lang/en/docs/cli/create/), the `bin` executable is called by `yarn create`. Not the `start` script.